NBitcoin/BIP39/Wordlist.cs

```diff
@@ -347,6 +347,11 @@ public override string ToString()
 			return _Name;
 		}
 
+		public string[] GetWords()
+		{
+			return (string[])_words.Clone();

```

Comment:
Why not return an IEnumerable?

Reply:
There's an unspoken contract between developers that nobody should modify a returned IEnumerable, unless you were the one who returned it: )

Reply:
This method is overloaded version of an already-existing one and for that reason i think it should return the same type that the original method. 
Given we return a clone, we shouldn't care if the caller modifies its elements.

Reply:
It doesn't matter that it wraps an already existing one, using IEnumerable allows to modify later the method to, for example, return a List instead of an Array, without changing the API.